tests/conftest.py

```diff
@@ -1739,6 +1739,9 @@ def rgw_endpoint(request):
         string: external RGW endpoint
 
     """
+    if float(config.ENV_DATA["ocs_version"]) > 4.6:
+        log.info("RGW service should be exposed by default. Skipping exposure.")
+        return
```

Comment:
Please add the endpoint from RGW route to not break any test functionality.

Reply:
Wouldn't the change done in `constants.py` take care of it?
The constants now reflect the name of the route, and `get_credentials()` looks for the route via the constants, IIUC

Reply:
Fixed